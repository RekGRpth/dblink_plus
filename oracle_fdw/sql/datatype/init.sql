\! sqlplus -s system/manager@orcl @sql/datatype/init_ora.sql
