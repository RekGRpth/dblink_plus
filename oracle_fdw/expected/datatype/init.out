\! sqlplus -s system/manager@orcl @sql/datatype/init_ora.sql

Session altered.


Session altered.


Session altered.


Session altered.


Session altered.


Session altered.


Table truncated.


Table truncated.


Table truncated.


Table truncated.


Table truncated.


Table truncated.


Table truncated.


Table truncated.


Table truncated.


Table truncated.


Table truncated.


Table truncated.


Table truncated.


Table truncated.


Table truncated.


Table truncated.


Table truncated.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


1 row created.


PL/SQL procedure successfully completed.


PL/SQL procedure successfully completed.


1 row created.


PL/SQL procedure successfully completed.


PL/SQL procedure successfully completed.


1 row created.


Grant succeeded.

